src/auth/dto/login.dto.ts

```diff
@@ -1,5 +1,5 @@
-import { IsString, Length } from "class-validator"
-import { ObjectId } from "mongoose"
+import { IsString, Length } from 'class-validator'
+import { ObjectId } from 'mongoose'
 
 export class LoginDto {
```

Comment:
Where is this used?

Reply:
I will double check later but I think you are right...this is probably dead code

Reply:
removed